manifests/charts/cloudcore/values.yaml

```diff
@@ -45,7 +45,7 @@ cloudCore:
       websocket:
         enable: true
       quic:
-        enable: false
+        enable: true
```

Comment:
Yes, during keadm init, the value of value. yml in help can be specified through -- set. However, now that the user has added a hubType during keadm join, is it appropriate for Cloudcore to also open quic by default? If the user does not configure quic to be opened through set and the keadm join specifies hubType as false, it will

Reply:
> The default value can be false, we can use flag `--set cloudCore.modules.cloudHub.quic=true` to enable quic

Yes, during keadm init, the value of value. yml in help can be specified through -- set. However, now that the user has added -- hub type during keadm join, is it appropriate for Cloudcore to also open quic by default? If the user does not configure it through set and the keadm join specifies -- hub type as false, an error may be reported.

Reply:
> Yes, during keadm init, the value of value. yml in chart can be specified through -- set. However, now that the user has added -- hub type during keadm join, is it appropriate for Cloudcore to also open quic by default? If the user does not configure quic through set and the keadm join specifies -- hub type as quic, an error may be reported.

Yeah, that makes sense :-)

Can we enable both websocket service and quic service in cloudcore? @fisherxu

Reply:
> > Yes, during keadm init, the value of value. yml in chart can be specified through -- set. However, now that the user has added -- hub type during keadm join, is it appropriate for Cloudcore to also open quic by default? If the user does not configure quic through set and the keadm join specifies -- hub type as quic, an error may be reported.
> 
> Yeah, that makes sense :-)
> 
> Can we enable both websocket service and quic service in cloudcore? @fisherxu

Cloudcore supports opening both quic and websocket protocols simultaneously.